pkg/cmd/roachtest/tests/backup.go

```diff
@@ -379,6 +379,9 @@ func registerBackup(r registry.Registry) {
 			CompatibleClouds:  registry.Clouds(cloudProvider),
 			Suites:            registry.Suites(registry.Nightly),
 			Run: func(ctx context.Context, t test.Test, c cluster.Cluster) {
+				if cloudProvider == spec.AWS {
```

Comment:
You could just remove the `cloudProvider` `for` loop? Or if you want to keep the AWS code around for now, I think the better option is to `range` over `[]string{spec.GCE}` so that we make use of the `CompatibleClouds` field.

For context, setting `CompatibleClouds` means that the test will be filtered out immediately if it's not compatible with the cloud provider the suite is testing. Doing a `t.Skip` means that roachtest will create a cluster for the purposes of running this test, only to have it skip itself immediately. That's a lot slower and more wasteful.

Reply:
Done.